USE-CASES/README.md

```diff
@@ -4,23 +4,103 @@ This directory contains the use cases for the next revisions of the WoT specific
 
 ## How to submit use cases
 
-1. Please describe new **use cases** useing the [Markdown template](use-case-template.md) [HTML template](use-case-template.html). (The HTML template is preferred.)
-2. Please submit your use cases to [this directory](https://github.com/w3c/wot-usecases/tree/main/USE-CASES) using [Pull Request](https://github.com/w3c/wot-usecases/pulls).
+Please submit your **use case** via [Github Issues](https://github.com/w3c/wot-usecases/issues/new/choose).
 
-## After approval by the Use Cases TF
+***Note:*** Please refer to the [notes below](#notes-on-the-items-within-the-use-case-template) when you describe your use case.
 
-1. Use cases that have been reviewed and approved for inclusion in [the IG note](https://w3c.github.io/wot-usecases/) are moved to https://github.com/w3c/wot-usecases/tree/master/USE-CASES/processed for archival purposes.
+## Review by the Use Cases TF
 
+1. The proposed use case will be reviewed by the Use Cases TF.
+1. If the use case is appoved, the Use Cases TF will include the use case into the [WoT Use Cases and Requirements Note](https://w3c.github.io/wot-usecases/).
+1. If the use case has any problems, e.g., need further clarification about the items, the Use Cases TF will give a comment on the GitHub Issue of the use case.
 
-# Terminology
+## Notes on the items within the Use Case Template
+
+### Submitter
+
+Put your name here
+    
+### Motivation
+
+Provide a description of the problem that is solved by the use case and a reason why this use case is important for the users
+    
+### Description
+
+Provide a description from the users perspective
+
+### Expectations for stakeholders and environments
+
+#### Target Users
+
+List all stakeholders that are involved in the use case from the following list:
+- device owners
+- device user
+- cloud provider
+- service provider
+- device manufacturer
+- gateway manufacturer
+- network operator (potentially transparent for WoT use cases)
+- identity provider
+- directory service operator
+
+#### Expected Devices
+
+List the target devices, e.g. as a sensor, solar panel, air conditioner
+
+#### Expected Data
+
+List the type of expected data, e.g. weather and climate data, medical conditions, machine sensors, vehicle data
+
+### Relationships with existing resources and standards
+
+#### Dependencies on WoT- Affected WoT deliverables and/or work items
+
+List the affected WoT deliverables that have to be changed to enable this use case
+
+
+#### Existing standards by other WGs within W3C
+
+Provide links to relevant standards that are relevant for this use case
+
+#### Existing standards outside W3C
+
+Provide links to relevant standards that are relevant for this use case
+
+### Optional clarificaitons on user's needs (optional but to be described as specific as possible so that requirements can be extracte precisely later)
```

Comment:
```suggestion
### Clarifications of user's needs (optional, but to be described as specifically as possible, so that precise requirements can be extracted later)
```